csharp/test/Drivers/Interop/Snowflake/DriverTests.cs

```diff
@@ -83,12 +83,6 @@ public DriverTests()
             Dictionary<string, string> options = new Dictionary<string, string>();
             _snowflakeDriver = SnowflakeTestingUtils.GetSnowflakeAdbcDriver(_testConfiguration, out parameters);
 
-            string databaseName = _testConfiguration.Metadata.Catalog;
```

Comment:
everything passes with these values removed?

Reply:
The database name is not required for the getting the metadata anymore. The calls prefix the specific DB name whose INFORMATION_SCHEMA tables need to be accessed.